frontend/components/api/Blocks.js

```diff
@@ -101,6 +101,49 @@ const Blocks = {
       console.error(error);
       throw error;
     }
+  },
+
+  getNextBlocks: async (lastBlockHeight, limit = 15) => {
```

Comment:
Am I reading this right that you basically implement a class/namespace for the functions? Why do you need the `const Blocks`? Can these be plain functions or defined as methods of a `Blocks` class?